examples/redux/package.json

```diff
@@ -8,14 +8,14 @@
 		"@axa-fr/react-oidc-redux": "^1.3.3",
 		"@axa-fr/react-oidc-redux-fetch": "^1.3.3",
 		"oidc-client": "^1.5.1",
-		"react": "^16.4.2",
-		"react-dom": "^16.4.2",
-		"react-redux": "~5.1.1",
-		"react-router-dom": "^4.3.1",
-		"recompose": "~0.30.0",
-		"redux": "^4.0.0",
-		"redux-oidc": "^3.0.0",
-		"redux-thunk": "^2.3.0"
+		"react": "^16.8.3",
+    "react-dom": "^16.8.3",
+    "react-redux": "~5.1.1",
```

Comment:
you still need redux ?

Reply:
We still have the redux version of the package, and in the redux example we need Redux for this.